### ISSUE TITLE
Quick fix: remove spaces in smtp server name

### ISSUE THREAD

**Issue by Te-k** [sandstorm-io/sandstorm#2640]
Hi there,

I got stock for one hour for a stupid space added in the end of the smtp server name.It would be nice to strip this variable :) (could save a lot of time to dump people like me)

The log error just in case : 
```
Exception while invoking method 'createAndEmailTokenForUser' Error: getaddrinfo ENOTFOUND [SERVER]  [SERVER] :25
    at Object.Future.wait (/programs/server/node_modules/fibers/future.js:449:15)
    at smtpSend (imports/server/email.js:74:44)
    at rawSend (imports/server/email.js:83:5)
    at send (imports/server/email.js:172:3)
    at sendTokenEmail (server/accounts/email-token/token-server.js:208:3)
    at _createAndEmailTokenForUser (server/accounts/email-token/token-server.js:261:3)
    at [object Object].createAndEmailTokenForUser (server/accounts/email-token/token-server.js:279:18)
    at maybeAuditArgumentChecks (packages/ddp-server/livedata_server.js:1711:12)
    at packages/ddp-server/livedata_server.js:711:19
    at [object Object]._.extend.withValue (packages/meteor/dynamics_nodejs.js:56:1)
    - - - - -
    at errnoException (dns.js:27:10)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:78:26)
```

**Comment by zarvox** [comment]
Looks like I trimmed the string in the [admin panel](https://github.com/sandstorm-io/sandstorm/blob/master/shell/client/admin/email-config-client.js#L16), but not the [setup wizard](https://github.com/sandstorm-io/sandstorm/blob/master/shell/client/setup-wizard/wizard.js#L567).  Nice catch!

**Comment by Te-k** [comment]
Not really, spent one hour checking my DNS config in all sense. Was fun though :p

**Comment by kentonv** [comment]
Fixed by #2647, to be released this weekend.
